core/src/main/java/com/linecorp/armeria/client/DefaultDnsQueryLifecycleObserver.java

```diff
@@ -61,7 +60,7 @@
     public void queryWritten(InetSocketAddress dnsServerAddress, ChannelFuture future) {
         meterRegistry.counter(meterIdPrefix.name().concat(".written"),
                 Arrays.asList(Tag.of(NAME_TAG, question.name()),
-                        Tag.of(SERVER_TAG, dnsServerAddress.getAddress().getHostAddress()))).increment();
+                        Tag.of(SERVERS_TAG, dnsServerAddress.getAddress().getHostAddress()))).increment();
```

Comment:
You have to use `SERVER_TAG` here because this tag is for a single name server. :wink: